### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Beelover05** [SeriousCreeper/Into-The-Betweenlands-2#30]
**Describe the bug**
Killing infestations causes you to get the untrue pacifist advancement (Could be that an entity was improperly tagged)

**To Reproduce**
Break apart an infestation and then let them recombine into one again

**Expected behavior**
Not getting an achievement? There was absolutely no way an innocent creature was harmed

**Version of the pack:**
Into the Betweenlands 2 Alpha-0.7.1

**Comment by Beelover05** [comment]
wait, they were neutral?

**Comment by sparoth** [comment]
yes they are neutral if you are a pacifist, as listed in the pacifist quest :)

**Comment by SeriousCreeper** [comment]
yes they are passive as pacifist, therefore killing them will break it.
